### ISSUE TITLE
%VotingPlugin_MilestoneCount% didn't work

### ISSUE THREAD

**Issue by hiiamken** [Ben12345rocks/VotingPlugin#1298]
**Versions**
Plugin version 6.14, Spigot version: Skyblock(Paper 1.19.4) - Survival(Paper 1.20.1)

**Relevant information**
[Configs that relate to the issue in a pastebin.](https://controlc.com/6ef49960)

**Context**
I am the person you assisted this morning. However, I encountered an issue where when players vote, the %VotingPlugin_MilestoneCount% does not increase (applies to both servers)

**Comment by BenCodez** [comment]
Have you done a restart since using the placeholder?

**Comment by hiiamken** [comment]
yes i restarst 3 times but its didnt work

**Comment by hiiamken** [comment]
both 2 server placeholder didnt increase

**Comment by BenCodez** [comment]
Your using /papi parse me (placeholder)?

**Comment by hiiamken** [comment]
The command is used for what purpose?

**Comment by BenCodez** [comment]
to test placeholders

**Comment by hiiamken** [comment]
yes it worked. it show "51"

**Comment by BenCodez** [comment]
Does it increase?

**Comment by hiiamken** [comment]
No, it didn't increase after I voted; it's still at 51 from yesterday until now.

**Comment by hiiamken** [comment]
All players in my server are affected

**Comment by BenCodez** [comment]
Where are you using the placeholder?

**Comment by hiiamken** [comment]
<img width="649" alt="Ảnh chụp Màn hình 2023-09-06 lúc 07 31 17" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/cb459dc0-50c5-486f-be1c-5ec744034075">
I just voted, and it still shows as 51. I'm using Deluxemenu to create the /vote GUI

**Comment by hiiamken** [comment]
Before setting it up like yesterday, the placeholder was working fine. Only after setting it up for different server rewards did this issue occur

**Comment by BenCodez** [comment]
Is it just on one server or all of them?

**Comment by BenCodez** [comment]
It should update eventually on a background update

**Comment by hiiamken** [comment]
All of my servers are affected.

**Comment by hiiamken** [comment]
<img width="405" alt="Ảnh chụp Màn hình 2023-09-06 lúc 07 49 38" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/0f942c95-b77a-4f41-a846-704ac9c13711">

Is it because I added this part? It's for the SOCKET method, but I'm currently using the PLUGINMESSAGING method.

**Comment by hiiamken** [comment]
<img width="425" alt="Ảnh chụp Màn hình 2023-09-06 lúc 07 51 56" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/9c9f06e3-e8ec-4c8a-a93a-c011be447c4b">

Here is my backend server setup (bungeeconfig)

**Comment by BenCodez** [comment]
https://bencodez.com/job/VotingPlugin/ does this help?

Setting the sockets/ports doesn't make a difference if you set method to PLUGINMESSAGING

**Comment by hiiamken** [comment]
Now I have to install this version for the proxy, and the backend servers, right?

**Comment by BenCodez** [comment]
yes

**Comment by hiiamken** [comment]
ok wait me a second

**Comment by hiiamken** [comment]
Can I use the 'test vote' command to test it? I've run out of votes on the websites for today.

**Comment by BenCodez** [comment]
Use /votingpluginbungee vote (player) (site)

**Comment by hiiamken** [comment]
![Uploading Ảnh chụp Màn hình 2023-09-06 lúc 08.07.59.png…]()

Its still 51

**Comment by hiiamken** [comment]
<img width="662" alt="Ảnh chụp Màn hình 2023-09-06 lúc 08 08 41" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/988b9399-b605-4928-9100-fdfa7aa36ab2">

**Comment by BenCodez** [comment]
Send your config.yml

**Comment by hiiamken** [comment]
<img width="890" alt="Ảnh chụp Màn hình 2023-09-06 lúc 08 11 03" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/da65f7ba-a51c-41ab-bb4e-7b6c0f024da9">

**Comment by hiiamken** [comment]
<img width="343" alt="Ảnh chụp Màn hình 2023-09-06 lúc 08 12 20" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/9255bc5c-245d-4760-8349-2628b5857b40">

**Comment by BenCodez** [comment]
Send all your configs.

**Comment by hiiamken** [comment]
https://controlc.com/15c8691a 
here

**Comment by hiiamken** [comment]
https://controlc.com/3c8f0255
serverdata

**Comment by hiiamken** [comment]
https://controlc.com/070bf84d
special rewards

**Comment by hiiamken** [comment]
https://controlc.com/db7eb315
votesite

**Comment by BenCodez** [comment]
you can try this again: https://bencodez.com/job/VotingPlugin/ but milestonecount should match all time total unless you did something. I need more info on how exactly your testing. 

I'm pretty sure this is working since I know active servers using placeholders with a very similar setup.

**Comment by hiiamken** [comment]
Will the milestones be different on each server, or will they be the same?

**Comment by BenCodez** [comment]
Should be the same on all servers

**Comment by hiiamken** [comment]
<img width="637" alt="Ảnh chụp Màn hình 2023-09-06 lúc 08 26 10" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/f90d9bc2-5295-40eb-beef-7e77c1daef01">

It seems like it still hasn't increased. Do you need any additional information from me? I'll provide it to you

**Comment by hiiamken** [comment]
<img width="922" alt="Ảnh chụp Màn hình 2023-09-06 lúc 08 27 07" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/d5563f40-d3a9-480d-b7d1-1ba724979e40">

what means 56//27//14//5//56//56//0//100

**Comment by BenCodez** [comment]
According to that milestonecount is 56. Have you tried the last build I sent? Might update differently. If not try /ac backgroundupdate and see if that does anything

**Comment by hiiamken** [comment]
Where should I use it? Are you referring to the command /ac backgroundupdate?

**Comment by BenCodez** [comment]
Oh, mean /av backgroundupdate, on backend servers

**Comment by hiiamken** [comment]
Its show "updating..."

**Comment by BenCodez** [comment]
Try viewing placeholders after that. You are using the latest build I linked though right? It's different from what you downloaded before

**Comment by hiiamken** [comment]
Hmm let me try again

**Comment by hiiamken** [comment]
1077, right?

**Comment by BenCodez** [comment]
just the latest, so yeah

**Comment by hiiamken** [comment]
<img width="942" alt="Ảnh chụp Màn hình 2023-09-06 lúc 08 40 29" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/4d1e34cd-dc6b-4b7d-a0bc-a71cdddb0809">
 
I saw that in console

**Comment by hiiamken** [comment]
Do I need to remove the PlaceholderAPI expansion and then run '/papi ecloud download VotingPlugin' again?

**Comment by BenCodez** [comment]
You can delete the expansion, it's no longer needed

**Comment by hiiamken** [comment]
It's still showing 51 after I restarted. Now I'll try removing the PlaceholderAPI expansion and then reload to see what happens.

**Comment by BenCodez** [comment]
What does it show with /av user (player) viewdata and /av user (player) viewcache?

**Comment by hiiamken** [comment]
<img width="351" alt="Ảnh chụp Màn hình 2023-09-06 lúc 08 47 07" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/2d474f56-d852-4116-a92d-df5b4459367f">

**Comment by BenCodez** [comment]
Are all mysql tables hooked to the same database? Looks like it's pulling whatever the data is showing

**Comment by hiiamken** [comment]
https://controlc.com/ee0add6e

**Comment by hiiamken** [comment]
yes wait me a second

**Comment by hiiamken** [comment]
<img width="422" alt="Ảnh chụp Màn hình 2023-09-06 lúc 08 49 06" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/587d4111-4290-4023-afa9-681569e1f5f0">

Survival

**Comment by hiiamken** [comment]
<img width="569" alt="Ảnh chụp Màn hình 2023-09-06 lúc 08 49 32" src="https://github.com/BenCodez/VotingPlugin/assets/142149960/aca3ff72-1eeb-4c3a-8f89-71bdc10c1361">

Bungee

**Comment by hiiamken** [comment]
oh i saw ip dont correct

**Comment by BenCodez** [comment]
That will do it

**Comment by hiiamken** [comment]
Let me try again @@

**Comment by hiiamken** [comment]
Its working tks for your support
